builder/files/boot/user-data

```diff
@@ -68,5 +68,8 @@ runcmd:
   # Pickup the hostname changes
   - 'systemctl restart avahi-daemon'
 
+  # Give ownership of scripts
+  - 'chown publiclab.publiclab /home/publiclab/Peggy -R'
```

Comment:
Cool, will this often be necessary? Does chroot-script run as root, and could this in theory be put there? Thinking about how to learn from this example. Thanks!

Reply:
I predict it is necessary because at the time of git cloning (image build time), it is running as root.
Commands in `user-data` run on first boot (including user creation!). This is my understanding from seeing cloud-init documentation.

We'll see!